### PR TITLE
Revert the changes of moving common data models to interface file for externalAppCardAction(ForCEA) files

### PR DESCRIPTION
**Pull request by MengyiGong**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Revert the changes of moving common data models to interface file for externalAppCardAction(ForCEA) APIs to avoid the breaking changes.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### CODE REVIEW

**Comment by AE-MS** [comment]
If you are reverting the change that was meant to avoid breaking changes, does that mean that we're now intentionally making a breaking change? Why did we want to avoid it before but now we are okay with it?

File: change/@microsoft-teams-js-4fe5ca49-b46c-4a6b-b580-bd6035df67e6.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Keep the common data models in the `externalAppCardAction` API file to share with the `externalAppCardActionForCEA` API.",
```
Comment (AE-MS) [inline change/@microsoft-teams-js-4fe5ca49-b46c-4a6b-b580-bd6035df67e6.json:3]: >Keep the common data models in the `externalAppCardAction` API file to share with the `externalAppCardActionForCEA` API. [](http://example.com/codeflow?start=14&length=120)

Consider:
"Unified common data models for external card actions into `externalAppCardActions` namespace"

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

**Comment by MengyiGong** [comment]
> If you are reverting the change that was meant to avoid breaking changes, does that mean that we're now intentionally making a breaking change? Why did we want to avoid it before but now we are okay with it?
From the changes in the last PR, we did not realize we are making a "breaking change" (all the data models and functions are marked with `@beta`, so it is not breaking). It was not intentional change.
Confirmed with @lakhveerk and @erinha please correct me if I'm wrong:
We believe MChat have not consumed the beta version from last week. We are fixing it before they consume our new version.

File: change/@microsoft-teams-js-4fe5ca49-b46c-4a6b-b580-bd6035df67e6.json (hunk 1 above)
Comment (MengyiGong) [reply change/@microsoft-teams-js-4fe5ca49-b46c-4a6b-b580-bd6035df67e6.json]: Updated with your suggestion. Thanks!

**Review by lakhveerk — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Review by jadahiya-MSFT — APPROVED** [review]

**Pull request merged**